cmd/rb-ldap/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/redbrick/rbldap/internal/pkg/rbldap"
+	"github.com/urfave/cli"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "rb-ldap"
+	app.Usage = "Command line interface for Redbrick Ldap"
+	app.ArgsUsage = ""
+	app.HideVersion = true
+	app.EnableBashCompletion = true
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "user, u",
+			Value: "cn=root,ou=ldap,o=redbrick",
+			Usage: "ldap user, used for authentication",
+		},
+
+		cli.StringFlag{
+			Name:  "dcu-user",
+			Value: "CN=rblookup,OU=Service Accounts,DC=ad,DC=dcu,DC=ie",
+			Usage: "Active Directory user for DCU, used for authentication",
+		},
+
+		cli.StringFlag{
+			Name:  "host",
+			Value: "ldap.internal",
+			Usage: "ldap host to query",
+		},
+
+		cli.StringFlag{
+			Name:  "dcu-host",
+			Value: "ad.dcu.ie",
+			Usage: "DCU Active Directory host to query",
+		},
+
+		cli.IntFlag{
```

Comment:
I think it would be better to have one flag for `host:port` both for RB and DCU

Reply:
I'm not a fan of a single flag as host and port won't change often and its more likely for just one too change then both. aswell in the library, they are handled separately

Reply:
I think @CSmartt input on this would be good

Reply:
`ldap.Dial` expect them combined, and they are concatenated together here: https://github.com/redbrick/userVhost/pull/7/files#diff-5bb0827606093060012c612e6c4acadbR20

Reply:
i still would rather have a separate flag for user and port

Reply:
+1 for separation, far more likely that one will change rather than both, if at all. Worth remembering also that we're not writing a generic LDAP tool here, so while I can see the case for a combined `host:port` flag, I think for clarity at our end we should keep them apart.